src/plugins/diagram-viewer/diagram-tile.tsx

```diff
@@ -56,20 +86,58 @@ export const DiagramToolComponent: React.FC<ITileProps> = observer((
 
   const toolbarProps = useToolbarTileApi({ id: model.id, enabled: !readOnly, onRegisterTileApi, onUnregisterTileApi });
 
+  const droppableId = `${kDiagramDroppableId}-${model.id}`;
+  const { isOver, setNodeRef } = useDroppable({ id: droppableId });
+  const dropTargetStyle = {
+    backgroundColor: isOver ? "#eef8ff" : undefined
+  };
+  useDndMonitor({
+    onDragEnd: (event: DragEndEvent) => {
```

Comment:
Nice! I think this is better than what we're currently doing in CODAP, so we may switch to this approach.